### DISCUSSION TITLE
Can we see some benchmarks ?

### DISCUSSION THREAD

**Discussion by dgandhi17** [isar/isar#18] [Help]
Can we see some benchmarks for millions of objects ?

**Comment by simc** [comment]
I will not compare different databases but I will provide some benchmarks.

But this is very low on my priority list. First I need to write a lot of unit tests 😬

**Reply by dgandhi17** [reply]
Sounds good, but before anyone try to use it they sure wanted to see some benchmarks, even though its not a comparison against others

**Reply by simc** [reply]
It is more that fast enough 😉
